ogc_ws/src/despatcher/scripts/regular.py

```diff
@@ -36,8 +36,11 @@ def __str__(self):
 # The regular payload should comprise only of short integers/characters
 # with the exception of the first (R msg prefix) and last (Unix timestamp)
 # Everything is standardized to big endian to keep in line with Rock 7's requirements
-# Example payload: r 1 1 30 226 1 30 2 2315 102 6857 4 0 0 1591089280
-struct_cmd = "> s B B B H B B B H B H B H B I"
+# Entries:         0 1 2 3  4   5 6  7  8 9 10 11   12  13   14 15 16 17 18
```

Comment:
What is entry 15 and could it potentially be packed as a char instead of unsigned short?

Note that the regular payload is sent over with all the entries listed in alphabetical order

Reply:
My apologies the unsigned hsort is supposed to be entry 16, namely waypoint. Entry 15 should be char